sdk/textanalytics/Azure.AI.TextAnalytics/src/PiiEntity.cs

```diff
@@ -23,6 +23,16 @@ internal PiiEntity(Entity entity)
             Length = entity.Length;
```

Comment:
Note to self: update so that this delegates to the component constructor and removes duplication.